Minuit/README.md

```diff
@@ -0,0 +1,15 @@
+This was synced with the Minuit2 source from ROOT using the standalone build.
```

Comment:
There should be a script to automate this, like the deleted `import_minuit.py`.

Reply:
The information should be placed in the iminuit docs folder, to avoid accidental loss.